roles/setup/tasks/main.yml

```diff
@@ -13,6 +13,8 @@
 
 - name: Install python packages
   ansible.builtin.include_tasks: python_packages.yml
+  when:
```

Comment:
im not a very big fan of this change...

this will silently make "additional_python_package" useless when "receptor_install_method" != 'package'

but alternative is that we will always install pip... /shrug

Reply:
maybe we can do something about how receptorctl is added to the additional_python_packages

Reply:
Useless `additional_python_package` here is intentional. If the `install_method` is `package`, everything should come from distribution packages imo, not from a mix of packages and and pip installs